CODEOWNERS

```
@@ -40,6 +40,9 @@
 /po/ @michaelchirico
 /R/translation.R @michaelchirico
 /src/po.h @michaelchirico
+/po/*zh_CN.po @Rdatatable/chinese

```

WDYT about this, to notify all translators whenever the templates are updated?

```suggestion
/po/*.pot @Rdatatable/translators
/po/*zh_CN.po @Rdatatable/chinese
```

---

great idea